<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopWindow</class>
 <widget class="QWidget" name="ShopWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>503</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>System</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Shop</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QListView" name="catListView">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>30</y>
         <width>591</width>
         <height>431</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QFrame" name="itemFrame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="addToCart_2">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>0</y>
             <width>31</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
          <widget class="QLabel" name="item_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
          <widget class="QLabel" name="price_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>190</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Price: </string>
           </property>
          </widget>
          <widget class="QLabel" name="goods_2">
           <property name="geometry">
            <rect>
             <x>16</x>
             <y>32</y>
             <width>151</width>
             <height>141</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>180</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QFrame" name="itemFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="addToCart">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>0</y>
             <width>31</width>
             <height>23</height>
            </rect>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
          <widget class="QLabel" name="item">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
          <widget class="QLabel" name="price">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>190</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Price:</string>
           </property>
          </widget>
          <widget class="QLabel" name="goods">
           <property name="geometry">
            <rect>
             <x>26</x>
             <y>32</y>
             <width>141</width>
             <height>141</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>180</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QFrame" name="itemFrame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="addToCart_3">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>0</y>
             <width>31</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
          <widget class="QLabel" name="item_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
          <widget class="QLabel" name="price_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>190</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Price:</string>
           </property>
          </widget>
          <widget class="QLabel" name="goods_3">
           <property name="geometry">
            <rect>
             <x>16</x>
             <y>32</y>
             <width>151</width>
             <height>141</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_3">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>180</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QFrame" name="itemFrame_4">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="addToCart_4">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>0</y>
             <width>31</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
          <widget class="QLabel" name="item_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
          <widget class="QLabel" name="price_4">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>190</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Price:</string>
           </property>
          </widget>
          <widget class="QLabel" name="goods_4">
           <property name="geometry">
            <rect>
             <x>16</x>
             <y>22</y>
             <width>161</width>
             <height>151</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_4">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>180</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QFrame" name="itemFrame_5">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="addToCart_5">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>0</y>
             <width>31</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
          <widget class="QLabel" name="item_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
          <widget class="QLabel" name="price_5">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>190</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Price:</string>
           </property>
          </widget>
          <widget class="QLabel" name="goods_5">
           <property name="geometry">
            <rect>
             <x>16</x>
             <y>32</y>
             <width>171</width>
             <height>141</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_5">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>180</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QFrame" name="itemFrame_6">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="addToCart_6">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>0</y>
             <width>31</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
          <widget class="QLabel" name="item_6">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>81</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
          <widget class="QLabel" name="price_6">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>190</y>
             <width>61</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Price:</string>
           </property>
          </widget>
          <widget class="QLabel" name="goods_6">
           <property name="geometry">
            <rect>
             <x>16</x>
             <y>22</y>
             <width>161</width>
             <height>151</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_6">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>180</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>301</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="category">
          <property name="text">
           <string>Category:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="catLabel">
          <property name="text">
           <string>CategoryName</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
